pkg/service/backup/config.go

```diff
@@ -37,6 +40,9 @@ func (c *Config) Validate() error {
 	if c.AgeMax < 0 {
 		err = multierr.Append(err, errors.New("invalid age_max, must be >= 0"))
 	}
+	if c.MaxLoadThreshold < 0 {
```

Comment:
Can the `MaxLoadThreshold` be `101` and higher? 

I would like to be sure that this validation is in sync with the condition we have in `worker_clustercheck.go`:
```go

func (w *worker) CheckCluster(ctx context.Context, hosts []hostInfo, clusterCheckBackoff retry.Backoff) error {
    ...
	if w.Config.MaxLoadThreshold == 100 {
		return nil
	}
    ...
}
```

Where we have exact equality with `100` hardcoded.

Also, do we want to consider `0` as a valid value in the config for disabling the repairs? If yes, I'll appreciate documenting such a fact in the config's field.

Reply:
Good point.
Let me change the check.

Reply:
About documenting 0 value.
Is this value for load even possible ? I think it's rather obvious that 0% disables the backup/repair.

Reply:
I would say that setting some field to 0 in config file should disable this option and not take the value 0 literally, especially when 0 value does not really make any sense.

For example, if backup's config `AgeMax` field is set to 0, it means that any backup run can be resumed, no matter how old it is, when if `AgeMax` field was taken literally, it should mean that no backup run can be resumed.
The same goes for `DiskSpaceFreeMinPercent` field, but in this case 0 taken literally and as disabling option has the same effect.

So I would propose that setting `MaxLoadThreshold` to 0 and 100 should have the same effect and just allow repair to run with any load.

Reply:
Otherwise, I would suggest that setting `MaxLoadThreshold` field to 0 in config should result in validation error - something is probably really wrong if someone is trying to schedule repair that always fails.

Reply:
Also, I would add some validation in terms of the upper bound of `MaxLoadThreshold`, because setting it to 200 also does not make any sense and I could indicate some form of an error.

Reply:
> I would say that setting some field to 0 in config file should disable this option and not take the value 0 literally, especially when 0 value does not really make any sense.

That's true, but frankly speaking setting MaxLoadThreshold to a bit higher value, e.g. **1** have almost the same effect.
Besides, it's float, you can set 0.0001 etc..
It's still > 0, but the effect is that there will be almost no backups/repairs taken.
In my understanding, setting some config param to 0 disables the feature only if we provide binary value to choose from. This is not the case here.

The value defines **maximal load threashold on nodes** that is OK to start repair/backup tasks. It's up to the end user to set this value he wants.

Default is 100 = no check.

Let me add explanation around this config field to the official scylla manager documentation, so that's it's clear what this value means.
As an addition, I'll add warning to config validation that will LOG WARNING when the value is low (let's say < 10).

I'm OK with adding validation error on value > 100 tough.